azure_functions_worker/testutils_lc.py

```diff
@@ -129,6 +134,12 @@ def _find_latest_mesh_image(cls,
         cls._mesh_images[host_major] = image_tag
         return image_tag
 
+    @staticmethod
+    def _download_azure_functions() -> str:
+        with urlopen(_FUNC_GITHUB_ZIP) as zipresp:
+            with ZipFile(BytesIO(zipresp.read())) as zfile:
+                zfile.extractall(tempfile.gettempdir())
```

Comment:
will the unzipped files in temp folder be automatically deleted after the test? if not, we need to explicitly delete them

Reply:
They tempfile should get deleted since its in the temp dir. We do the same in setup.py

Reply:
i found you need to turn on auto delete setting otherwise temp files can accumulate: https://www.easeus.com/data-recovery-solution/automatically-delete-temp-files.html, is that taken care in the pipeline?

Reply:
It should be removed since it a tmp file but to be safe I added an update to delete the file